Add PromMetrics to collect live data from a Prometheus Host

4n4nd opened the pull request:
Added PromMetrics class which acts like an observble.
Added prometheus.py as a connector to a Prometheus Host.
Updated Pipfile to add apscheduler as a dependency.
Updated Pipfile.lock.

Commit:
Add PromMetrics to collect live data from a Prometheus Host

Added PromMetrics class which acts like an observble.
Added prometheus.py as a connector to a Prometheus Host.
Updated Pipfile to add apscheduler as a dependency.
Updated Pipfile.lock.

42B commented on metrics/prommetrics.py:
```diff
@@ -0,0 +1,57 @@
+    def _get_metrics_from_prometheus(self, observer):
+        # Collect credentials to connect to a prometheus instance
+        prom_token = os.getenv("PROM_ACCESS_TOKEN",None)
+        prom_url = os.getenv("PROM_URL",None)
```
```python
prom_token = os.getenv("PROM_ACCESS_TOKEN")
prom_url = os.getenv("PROM_URL")
```
?

42B commented on metrics/prommetrics.py:
```diff
@@ -0,0 +1,57 @@
+        prom_url = os.getenv("PROM_URL",None)
+        if not (prom_token or prom_url):
+            print("Prometheus credentials not found")
+            exit(0)
```
Shouldn't this use `sys.exit` and also return a nonzero value?

4n4nd replied:
Good idea, I will change it.
Thanks

Commit:
Add system exit error on no credentials

Added an error text for system exit when no prometheus credentials are provided.
added a return value for _get_metrics_from_prometheus()

durandom reviewed (approved):
That looks great. 
Awesome work @4n4nd
